provider/kubernetes/builder_ingress_test.go

```diff
@@ -92,6 +92,23 @@ func iBackend(name string, port intstr.IntOrString) func(*v1beta1.HTTPIngressPat
 	}
 }
 
+func iTLSs(opts ...func(*v1beta1.IngressTLS)) func(*v1beta1.Ingress) {
```

Comment:
Can we rename this to `iTLSes`? Makes it slightly more readable (and grammatically correct) IMHO.